src/store/modules/TsModsModule.ts

```diff
@@ -118,6 +120,27 @@ export const TsModsModule = {
             return getters.cachedMod(mod).isLatest;
         },
 
+        /*** Was the last successful mod list update more than an hour ago? */
+        isModListOutdated(state) {
+            return state.modsLastUpdated instanceof Date
+                && (Date.now() - state.modsLastUpdated.getTime()) > (1000 * 60 * 60);
+        },
+
+        /*** A more concise version of the error message */
+        conciseThunderstoreModListUpdateErrorMessage(state): string|undefined {
+            if (!state.thunderstoreModListUpdateError) {
+                return undefined;
+            }
+
+            let conciseError = "Failed to load mod list";
+            if (isNetworkError(state.thunderstoreModListUpdateError)) {
+                conciseError = "Failed to download mod list due to network error"
```

Comment:
Could be "Failed to fully refresh the online mod list due to network error" to be consistent (if screen estate allows).

Reply:
Now that you mentioned this, it reminded me that I was also contemplating if the StatusIndicator's text should be a warning or an error (yellow vs. red text), but that's also definitely out of the scope of this PR. :D

Reply:
It's a partial failure so I'd say it's an error. Especially since some users seem to be confused about the status unless we're very in-your-face about it.